### PR TITLE
kotlin-native: Update to version 1.3.11

### PR DESCRIPTION
**Pull request by Arthur-Tesse**

### CODE REVIEW

File: bucket/kotlin-native.json [hunk 1]
```diff
@@ -20,11 +21,11 @@
         ]
     },
     "checkver": {
-        "github": "https://github.com/JetBrains/kotlin-native"
+        "github": "https://github.com/JetBrains/kotlin"
     },
     "autoupdate": {
         "url": "https://github.com/JetBrains/kotlin-native/releases/download/v$version/kotlin-native-windows-$version.zip",
```
Comment (Ash258) [inline bucket/kotlin-native.json]: Still old url 

```diff
-https://github.com/JetBrains/kotlin-native/releases/...
+https://github.com/JetBrains/kotlin/releases/...
```

File: bucket/kotlin-native.json [hunk 2]
```diff
@@ -2,11 +2,12 @@
     "homepage": "https://kotlinlang.org/",
     "version": "0.9.3",
```
Comment (rasa) [inline bucket/kotlin-native.json]: Shouldn’t the version bump to 1.3.11 too?

Comment (Arthur-Tesse) [reply bucket/kotlin-native.json]: My bad... Missed this field

**Comment by Ash258** [comment]
Autoupdate is still missing. Fix autoupdate property, then run checkver to update all fields automatically.

**Comment by Arthur-Tesse** [comment]
Autoupdate is working fine with me and checkver too ! I used this command to check
`PS > .\bin\checkver.ps1 kotlin-native`
`kotlin-native: 1.3.11`

**Comment by Ash258** [comment]
Update will fail if you run it with `-Force` parameter, becase autoupdate url is set to use old repository.

**Pull request merged**